src/erlang_ls_completion_provider.erl

```diff
@@ -208,3 +218,17 @@ include_file_macros(Kind, Name) ->
     {error, _} ->
       []
   end.
+
+%%==============================================================================
+%% Keywords
+%%==============================================================================
+
+-spec keywords() -> [map()].
+keywords() ->
+  Keywords = [ 'after', 'and', 'andalso', 'band', 'begin', 'bnot', 'bor', 'bsl'
```

Comment:
Why not have them as binaries in the list already?

Reply:
I was going to do that, but then decided not to for readability.